### ISSUE TITLE
support for Razer Deathstalker V2 Pro

### ISSUE THREAD

**Issue by simonedelpopolo** [1kc/razer-macos#805]
Hello,
is it possible to add support for this device?
thank you.

**Comment by j-lepard** [comment]
Im in the same boat - but I am a bit confused as the DSV2 is listed in the supported devices.

**Comment by simonedelpopolo** [comment]
I have compared the code for linux and macOS and many features are missing from macOS code.
It doesn't seems big deal to fix it. For the once knowing C :D

waiting for the new release if any.
thanks.

**Comment by j-lepard** [comment]
I am very very eager to simply 'turn off' the rgb on my keyboard. Anything else would be amazing. 
I've gone so far as to look for a comprable (and supported) keyboard in the linux distribution then make the appropriate additions to the razer-macos repository. From there, I had to figure out how to compile the files to build the application. Then I could at least test it. 

After much faffing about about I actualy did manage to added the lines to the code, and build the app. Admitedly, this is not  hard for people who actually know what they are doing. But it was a first for me. 

Yay! The app opened!
Boo!!!! it did not recognize my keyboard. 

I kinda gave up and pursued OpenRGB as it seemed more active. 
That said... it also works on pretty much everything except my mac. 

I have resorted to using a hateful compact chiclet keyboard again. 

grrrrr.

**Comment by simonedelpopolo** [comment]
thank you for OpenRGB hint, I didn't know it existed

**Comment by j-lepard** [comment]
Yes.. but dont get too excited - It (keyboard) isnt available there either. BUT, the good news is, there are a couple of guys actively working on it. Fingers crossed...

**Comment by simonedelpopolo** [comment]
just installed and I can set keyboard keys selectively and save the profile.

**Comment by j-lepard** [comment]
Ohhh interesting. May I ask what version of osx you are running?

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by simonedelpopolo** [comment]
yes sure,

Ventura 13.5.1 Mac mini M1

**Comment by LolKekkk** [comment]
Great job!

**Comment by rsrzrkbrd** [comment]
Hi, struggling with the Deathstalker v2 Pro keyboard. Anyone working on making this app compatible with that device? Would save this inordinate investment in a keyboard... ! Thanks
